CONTRIBUTING.md

```diff
@@ -9,7 +7,68 @@ MongoDB welcomes community contributions! If you’re interested in making a con
+4. Commit and push your changes to your branch then submit a pull request against the current release branch, not master.  The naming scheme of the branch is `release-staging-v#.#.#`. Note: There will only be one release branch at a time.  
+5. A repo maintainer will review the your pull request, and may either request additional changes or merge the pull request.
+
+## Autoclose stale issues and PRs
+- After 30 days of no activity (no comments or commits are on an issue or PR) we automatically tag it as “stale” and add a message: "This issue has gone 30 days without any activity and meets the project’s definition of ‘stale’. This will be auto-closed if there is no new activity over the next 30 days. If the issue is still relevant and active, you can simply comment with a “bump” to keep it open, or add the “[Status] Not Stale” label. Thanks for keeping our repository healthy!"
+- After 30 more days of no activity we automatically close the issue / PR.
+
+## Prerequisites
+- [AWS CloudFormation CLI](https://github.com/aws-cloudformation/cloudformation-cli)
+- (Optional - only need if building from source) [AWS CloudFormation CLI Go Plugin](https://github.com/aws-cloudformation/cloudformation-cli-go-plugin/) > v1.0
+- (Optional - only need if building from source) [Go](https://golang.org/doc/install) v1.19
+
+## Testing the Provider
+Please see README for each resource for details on unit and integrated AWS testing.
+
+## Securing your Deployment
+This repository contains utilities which use both the AWS and MongoDB Atlas APIs to progamatically create, destory, or otherwise manage resources in the AWS and MongoDB clouds. This can have costs or other impact so please use these tools with diligence and follow best practices.
+
+The automated workflows require the following secrets to be placed into your Github forks' Settings/Secrets:
+
+```
+AWS_ACCESS_KEY_ID
+AWS_SECRET_ACCESS_KEY
+AWS_REGION
+ATLAS_PUBLIC_KEY
+ATLAS_PRIVATE_KEY
+ATLAS_ORG_ID
+```
+
+## Security - Setup
+
+1. Create and note your MongoDB Atlas API Key.
+2. Create and note your AWS Access Key and AWS Secret Key ID.
+3. Follow the Github docs on how to [create a Secret](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#creating-encrypted-secrets-for-a-repository) in your clone of this repository.
+
+## Using the MongoDB Atlas CFN Resources
+
+The fastest way to use the resources is with the official [MongoDB Atlas on AWS Partner Solutions](https://aws.amazon.com/solutions/partners/mongodb-atlas/) CloudFormation templates. The [templates](https://github.com/aws-quickstart/quickstart-mongodb-atlas/templates) folder contains concrete CloudFormation templates you can use to start your own projects.
+
+There are two main parts of this project:
+
+1. [quickstart-mongodbatlas](quickstart-mongodbatlas) This is a mirror of https://github.com/aws-quickstart/quickstart-mongodb-atlas.
```

Comment:
404 error on this first link

Reply:
I just check there and the link is live - I know a different one failed before, maybe it was a caching issue on your side? 

Can you verify, just in case I'm missing something here?

Reply:
yes works now, likely an issue on my end